examples/create3_robots/cliff_sensors.py

```diff
@@ -0,0 +1,30 @@
+#
+# Licensed under 3-Clause BSD license available in the License file. Copyright (c) 2021-2022 iRobot Corporation. All rights reserved.
+#
+
+from irobot_edu_sdk.backend.bluetooth import Bluetooth
+from irobot_edu_sdk.robots import event, hand_over, Color, Robot, Root, Create3
```

Comment:
No need to include `Color` or `Root`

Reply:
I Just copied it from the other examples, otherwise I would of removed them. You might want to go check all the other examples for that as well. But I will remove it no problem.

Reply:
Agreed -- we are inconsistent -- not a judgement on you. Just trying to fix them as we touch files.

Reply:
Gotcha. Been there.